lib/tasks/import_metadata.rake

```diff
@@ -0,0 +1,68 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task :metadata, [:csv_file] => [:environment] do |t, args|
+
+    import_csv_file(args.csv_file)
+
+  end
+
+  def import_csv_file file
+
+    csv = File.open(file, encoding: "utf-8")
+
+    csv_headers = File.readlines(csv).first.split(",")
+
+    metadata_hash = {
+      category: csv_headers[0],
+      resource: csv_headers[1],
+      version: csv_headers[2],
+      contact_organisation: csv_headers[3],
+      dataset_id: csv_headers[4],
+      website_download_link: csv_headers[6],
+      metadata: nil,
+      factsheet: csv_headers[7],
+      marine_spatial_planning: csv_headers[8],
+      education: csv_headers[9],
+      environmental_impact_assessment: csv_headers[10],
+      ecosystem_assessment: csv_headers[11],
+      ecosystem_services: csv_headers[12]
+    }
+
+    CSV.parse(csv, :headers => true, encoding: "utf-8") do |row|
```

Comment:
I've just noticed this. Can you please use the latest ruby syntax?
So change `:headers => true` to `headers: true`

Reply:
@Levia thanks I have made this alteration.